client/src/containers/overview/table/utils.ts

```diff
@@ -1,9 +1,14 @@
+import { ProjectScorecardView } from "@shared/entities/project-scorecard.view";
 import { Project } from "@shared/entities/projects.entity";
 import { z } from "zod";
 
-import { filtersSchema } from "@/app/(overview)/url-store";
+import {
+  filtersSchema,
+  scorecardFiltersSchema,
+} from "@/app/(overview)/url-store";
 
 export const NO_DATA: Project[] = [];
+export const NO_SCORECARD_DATA: ProjectScorecardView[] = [];
```

Comment:
I don't want to have a constant every time we have a new table, so let's just keep `const NO_DATA = []` (without typing) and type it where you use it (`NO_DATA as Project[]`).